src/extension/background-script/migrations/index.ts

```diff
@@ -40,6 +41,18 @@ const migrations = {
     }
   },
 
+  migratePaymentsWithoutAccountId: async () => {
+    const { accounts } = state.getState();
+    if (Object.keys(accounts).length == 1) {
```

Comment:
hmm. the issue is that we then still have unexpected data in the DBs which we will always have to account for. 

can we associate the payments now, but not change the filtering and do that in an upcoming release. 
I assume old payments are less relevant

Reply:
I thought the same. 👍

>can we associate the payments now, but not change the filtering and do that in an upcoming release.

Currently the transaction list is not filtered by account, so that should already be the case.